Source/ACE.Common/DerethDateTime.cs

```diff
@@ -1059,6 +1061,6 @@ public static DerethDateTime ConvertRealWorldToLoreDateTime(DateTime dateTime)
         /// <summary>
         /// Converts the <see cref="DateTime.UtcNow"/> object to a new <see cref="DerethDateTime"/> object set to EMU Standard Sync Time.
         /// </summary>
-        public static DerethDateTime UtcNowToEMUTime => new DerethDateTime((DateTime.UtcNow - TimeZoneInfo.ConvertTimeToUtc(retailDayLast_RealWorld, TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time"))).TotalSeconds);
+        public static DerethDateTime UtcNowToEMUTime => new DerethDateTime((DateTime.UtcNow - TimeZoneInfo.ConvertTimeToUtc(retailDayLast_RealWorld, TZConvert.GetTimeZoneInfo("Eastern Standard Time"))).TotalSeconds);
```

Comment:
a specific timezone shouldn't be hardcoded into the emu like this.

if anything, this should be UTC.

Reply:
the more i look at this, the more it just seems wrong

who originally approved this pr?

answer: no one. and this is why we have code reviews, and this is why this practice of checking in prs without approval was supposed to be banned long ago...